### PR TITLE
feat: Implement MedRecordValue

### PR DESCRIPTION
**Pull request by JabobKrauskopf**

### CODE REVIEW

File: crates/medmodels-core/src/medrecord/values.rs [hunk 1]
```diff
@@ -0,0 +1,22 @@
+#[derive(Debug, Clone)]
+pub enum MedRecordValue {
+    String(String),
+    Int(i64),
+    Float(f64),
+    Bool(bool),
+}
+
+macro_rules! implement_from {
```
Comment (JabobKrauskopf) [inline crates/medmodels-core/src/medrecord/values.rs:9]: Rust is sooooooo amazing for enabling stuff like this

File: rustmodels/src/medrecord/conversion.rs [hunk 2]
```diff
@@ -0,0 +1,111 @@
...
+        match self.0 {
+            MedRecordValue::String(value) => value.into_py(py),
+            MedRecordValue::Int(value) => value.into_py(py),
+            MedRecordValue::Float(value) => value.into_py(py),
+            MedRecordValue::Bool(value) => value.into_py(py),
+        }
+    }
+}
+
+pub trait DeepFrom<T> {
```
Comment (JabobKrauskopf) [inline rustmodels/src/medrecord/conversion.rs:56]: Also checkout the magic implemented with `DeepFrom`

**Review by FloLimebit — CHANGES_REQUESTED** [review]
maybe we should add a little description in the readme that the installation of rust is needed if someone wants to install medmodels from source. Quick guide on how to install rust (basically two lines i guess)

**Comment by JabobKrauskopf** [comment]
> maybe we should add a little description in the readme that the installation of rust is needed if someone wants to install medmodels from source. Quick guide on how to install rust (basically two lines i guess)

@FloLimebit I agree but I would suggest that this should be addressed in a different issue/pr (https://github.com/limebit/medmodels/issues/22)

**Review by FloLimebit — APPROVED** [review]

**Pull request merged**